rotkehlchen/api/v1/schemas.py

```diff
@@ -3006,14 +3008,17 @@ def make_nft_filter_query(
             elif data['ignored_assets_handling'] == IgnoredAssetsHandling.SHOW_ONLY:
                 ignored_assets_filter_params = ('IN', self.db.get_ignored_asset_ids(cursor))
 
+        owner_addresses = data['owner_addresses']
+        if owner_addresses is None:
+            owner_addresses = self.chains_aggregator.queried_addresses_for_module('nfts')
```

Comment:
I believe this should go in one line as we do in other places

```suggestion
        owner_addresses = self.chains_aggregator.queried_addresses_for_module('nfts') if data['owner_addresses'] is None else data['owner_addresses']
```

Another option is

```suggestion
        if (owner_addresses := data['owner_addresses']) is None:
            owner_addresses = self.chains_aggregator.queried_addresses_for_module('nfts')
```